diff --git a/dist/index.mjs b/dist/index.mjs
index c543f3e83ffb5210495ac735f9a906e59b9da491..1220f691bffd449834e83811da0afbe92a8d1536 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1839,7 +1839,7 @@ const pluginWebAccessibleResources = () => {
           // all resources are web accessible
           resources: ["**/*", "*"],
           // change the extension origin on every reload
-          use_dynamic_url: true
+          use_dynamic_url: false
         };
         if (browser === "firefox") {
           delete war.use_dynamic_url;
